### PR TITLE
Feature/296 favicon fe

### PR DESCRIPTION
**Pull request by BrianXu20**
[Ticket # 296:](https://projects.torchbox.com/projects/tbxcom/tickets/296)

- Fixed mstitle-150x150.png not found issue
- Changed Mac OS EL Capitan Safari icon color to #FD5765

### CODE REVIEW

File: tbx/static_src/images/icons/safari-pinned-tab.svg [hunk 1]
```diff
@@ -1 +1,24 @@
-<svg version="1" xmlns="http://www.w3.org/2000/svg" width="438.667" height="438.667" viewBox="0 0 329.000000 329.000000"><path d="M141.1 8.3c-1.7 1.7-1.4 3.9 1.4 10.2 9.3 20.9 10.5 43.7 3.4 65-5.3 15.8-12.1 26.9-28.8 46.9-10.1 12.2-19.8 26.2-25.8 37.7l-4 7.5-2.6-4c-7.1-11.5-12.7-30.9-14.1-49-.3-3.8-.7-7-.8-7.1-.2-.2-1.3-.6-2.5-.9-1.7-.6-2.7.1-5.1 3.6-16.4 24-24 46.6-23.6 69.8.4 24.5 5.1 44.5 14.9 64 12.3 24.7 29.2 43 51.6 56.1 8.4 4.8 22.4 10.6 28.2 11.4 6 .9 6.5-.9 2.7-8.8-4.4-9.2-6.1-17-6.5-29.2-.4-12.5 1.8-23 8-39 5.9-15.1 21.3-43.7 29.3-54.3l2.4-3.2 4.8 7.2c13.1 20.3 24.3 42.5 29.6 58.7 1.3 4.1 2.7 8.8 3 10.5.3 1.7.9 4.8 1.3 6.8 1.3 5.6.6 21.4-1.3 29.3-.9 3.8-3.1 10.1-4.8 13.9-3 6.4-3.1 6.9-1.4 8.1 1.4 1.1 2.9.9 8.9-.9 33.2-10.4 60.2-34.1 76.6-67.1 10.3-20.8 14.1-36.8 13.5-57.8-.5-19.6-1.9-25.2-5.7-23.1-.8.5-4.2 5-7.6 10-3.3 5.1-8.7 11.8-11.8 15.1-5.9 6-13.1 11.9-13.7 11.2-.2-.2-.7-5.6-1-11.9-.4-6.3-.9-12.3-1.1-13.5-.2-1.1-.7-4.3-1.1-7-1-8-1.1-8.6-3.2-17.5-8-33.6-22.8-63.7-45-91.4-16.2-20.2-37.3-39.7-57.7-53.3-7.3-4.8-8.9-5.5-10.4-4z"/></svg>
\ No newline at end of file
+<?xml version="1.0" standalone="no"?>
```
Comment (SharmaineLim) [inline tbx/static_src/images/icons/safari-pinned-tab.svg:1]: Is this change needed?

Comment (BrianXu20) [reply tbx/static_src/images/icons/safari-pinned-tab.svg:1]: I think it's unnecessary but just in case, I copied the entire generated file.
